### DISCUSSION TITLE
Not able to open XML file

### DISCUSSION THREAD

**Discussion by ale19co79** [phcode-dev/phoenix#1752] [Q&A]
Hi,
I just installed the Phoneix on my Mac and I use it mainly to modfy SQL and XML file.
With SQL files I'm not having any issue. With the XML one I can open it only double clicking on the file if the software is not running or through the FILE menu clicking on Open file.
If I try to double clicking on the file with the software already running in background I get the following error:
![image](https://github.com/user-attachments/assets/0732ba61-7a34-448a-84a2-97651a604d1e)

Could you please help me understand why?
Many thanks
Alex

**Comment by abose** [comment]
Thanks for reporting this issue. I tested it out. I am not able to repro this. Step-by-step instructions on how to recreate this issue will help.

**Reply by ale19co79** [reply]
Hi Abose,
This is the step by step instructions:

1. So I open PHOENIX with a double click on the Icon
2. I open a foldrr where the can find the file 
3. Double click on the XML file
4. I'm getting the error and the file is not opened

Thank you

**Reply by abose** [reply]
Which version of mac os is this?
Mac m1 or intel.

**Reply by ale19co79** [reply]
![image](https://github.com/user-attachments/assets/4ccb7673-2f2a-487a-a099-fa552ce19255)

**Comment by ale19co79** [comment]
HI,
Anyone that could help me on that?

**Reply by abose** [reply]
Hi @ale19co79 i was unable to reproduce this issue in my Mac. If you have another mac, can you see if its a problem in that mac too.

**Comment by charlypa** [comment]
@ale19co79 
would you please share a sample XML file in which you face the iissue. That would help us to debug the issue further

**Comment by ale19co79** [comment]
Hi All,

first of all many thanks for your answer.
Don't know how, but seem that the issue is now fixed.

Thank you again for the support.
Alex
